README.md

```diff
@@ -72,7 +72,8 @@ Immutable([1, 2, 3]).concat([10, 9, 8]).sort()
 // [1, 2, 3, 4, 5, 6], because a vanilla array's concat() method has
 // no knowledge of Immutable.
 
-Immutable({all: "your base", are: {belong: "to them"}}).merge({are: {belong: "to us"}})
+var obj = Immutable({all: "your base", are: {belong: "to them"}});
+Immutable.merge(obj, {are: {belong: "to us"}})
 // This handy new method will return the following:
```

Comment:
It's not really a "method" per se anymore, so maybe the wording should be changed here